src/main/resources/internal-admin-api.yaml

```diff
@@ -167,7 +167,11 @@ paths:
             type: string
             description: "JWT is valid."
         400:
-          description: "JWT is invalid (e.g. expired)."
+          description: "JWT is invalid."
+        404:
+          description: "JWT not found (e.g. expired). Returns the initial request back to resume user journey."
```

Comment:
I know this was an old message but I'm not sure `e.g.` is the correct term here. (https://www.grammarly.com/blog/know-your-latin-i-e-vs-e-g/)
Maybe using `JWT not found or expired.` would be better.